feature/login/src/main/java/com/yapp/feature/login/LoginState.kt

```diff
@@ -1,15 +1,21 @@
 package com.yapp.feature.login
 
 
-sealed interface LoginState {
-    data object Loading : LoginState
-    data object LoginSuccess : LoginState
-    data object LoginError : LoginState
+data class LoginState(
+    val id: String = "",
+    val password: String = "",
+    val idFailDescription : String? = null,
+    val pwFailDescription : String? = null,
+    val isPWVisible: Boolean = false, // 비밀번호 표시 여부
+){
+    val isActivateLoginButton : Boolean = id.isNotEmpty() && password.isNotEmpty()
```

Comment:
```suggestion
    val idErrorDescription : String? = null,
    val passwordErrorDescription : String? = null,
    val isPasswordVisible: Boolean = false, // 비밀번호 표시 여부
){
    val enableLoginButton : Boolean = id.isNotEmpty() && password.isNotEmpty()
```

A: 어디는 PW고 어디는 password인게 살~짝 불편하군요
C: 기본 컴포넌트에서 error, enable 네이밍을 사용하고 있기 때문에 용어를 error, enable로 통일하는게 좋아보여요